iothub/service/src/DigitalTwin/Authentication/PnpSasCredential.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure;
+using Microsoft.Azure.Devices.Authentication;
+
+namespace Microsoft.Azure.Devices.DigitalTwin.Authentication
+{
+    internal class PnpSasCredential : IotServiceClientCredentials
+    {
+        private AzureSasCredential _credential;
+
+        public PnpSasCredential(AzureSasCredential credential)
+        {
+            _credential = credential;
+        }
+
+        public override string GetAuthorizationHeader()
+        {
+            return _credential.Signature;
+        }
```

Comment:
correct me if I'm wrong, was there also an "update" functionality with our iot hub sas credential type?

Reply:
That the user needs to implement to updated the signature. But we always grab the signature.